### ISSUE TITLE
Car lift not raising vehicle?

### ISSUE THREAD

**Issue by SDGNelson** [SmartlyDressedGames/Unturned-3.x-Community#4560]
Separated from #4557.

**Comment by Kubusiek3** [comment]
> Separated from #4557.

yes and it fall off when you touch it someway and falls down sameway with tank on #4557

**Comment by SDGNelson** [comment]
Checked this and fortunately (or unfortunately) it's the same reason this used to be an issue: The vehicle rigidbody is falling asleep almost immediately after the player exits, whereas in the past it used to be a bit longer before it fell asleep. There should be an open "known issue" for this but as there isn't I'll mark this as that.
